gluster/cli/peer.py

```diff
@@ -47,6 +47,34 @@ def detach(host):
     cmd = ["detach", host]
     return peer_execute(cmd)
 
+def detachAll():
+    """
+    Removes All Hosts from Cluster
+
+    :returns: nothing
+     print: success and failure messages, raises
```

Comment:
Since this will be used as library like 

    from gluster.cli import peer
    peer.detach_all()

Raise exceptions in case of failures and print nothing in case of success.